community-members.md

```diff
@@ -45,23 +45,19 @@ Members of the Technical Committee are the maintainers of
 [Project specs](https://github.com/open-telemetry/opentelemetry-specification)
 and [Proto definitions](https://github.com/open-telemetry/opentelemetry-proto).
 
-Trace Approvers:
+[Specification sponsors](./community-membership.md#specification-sponsor) are
```

Comment:
These are only initially seeded with our current specs approvers but @open-telemetry/specs-approvers continues to exist independently, so we still need to document them somewhere. Do we want to list them here in this document, or in https://github.com/open-telemetry/opentelemetry-specification/blob/main/README.md, or https://github.com/open-telemetry/opentelemetry-specification/blob/main/CONTRIBUTING.md?